src/main/java/com/aws/greengrass/certificatemanager/certificate/CertificateExpiryMonitor.java

```diff
@@ -22,6 +28,9 @@ public class CertificateExpiryMonitor {
     private static final Logger LOGGER = LogManager.getLogger(CertificateExpiryMonitor.class);
     private static final long DEFAULT_CERT_EXPIRY_CHECK_SECONDS = 30;
 
+    @Setter(AccessLevel.PACKAGE)
+    private Clock clock = Clock.systemUTC();
```

Comment:
clock should be injected from the Nucleus, add `Clock` to the constructor, system utc will be given

Reply:
done